fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -458,10 +459,10 @@ public DynamicPartitionScheduler getDynamicPartitionScheduler() {
     }
 
     private static class SingletonHolder {
-        private static final Catalog INSTANCE = new Catalog();
+        private static final Catalog INSTANCE = new Catalog(false);
     }
 
-    private Catalog() {
+    private Catalog(boolean isCheckpointCatalog) {
```

Comment:
maybe we should add some comment to this arg, such as: isCheckpointCatalog=true means we need to do some thread metrics collect.

Reply:
ok